us/en/skymiles/blocks/blade/blade.css

```diff
@@ -111,70 +125,48 @@
 }
 
 @media (min-width: 992px) {
-    .blade > div {
+    .blade > div,
+    .blade > div > a:any-link {
         flex-direction: unset;
         align-items: stretch;
     }
 
-    .blade {
-        max-width: 1200px;
-    }
-
     .blade .blade-image {
         position: relative;
-        flex: 5;
-        display: flex;
-        flex-direction: column;
-        align-items: center;
-        justify-content: center;
+        width: 30%;
```

Comment:
From the top of my head, the delta website is using a 12 cols grid system, which is why we initially used flex 4/8 or 5/7, so it tries to align properly with the grid. Using 30%/70% will typically not align with a 12 cols grid… you'd be on a 10 cols one.

I wouldn't really touch the styles as part of the this, aside from repeating the wrapper styles on the wrapping anchor so it behaves consistently